src/routes/posts.js

```diff
@@ -33,6 +34,49 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET REQUESTS
+
+// GET POST BY ID
+
+const getPost = async (req, res, next) => {
+    const { id } = req.params
+    try {
+        const posts = await db.query(
+            'SELECT p.*, jsonb_agg(c.* ORDER BY c."createdAt" DESC) as comments FROM posts p LEFT JOIN comments c ON p.id = c."postId" WHERE p.id=$1 GROUP BY p.id;',
+            [id]
+        )
+        const article = posts.rows[0]
+
+        if (!article) {
+            res.status(404).json({
```

Comment:
just me nitpicking. we can use a `return` here so we don't have to use an `else` on line `55` later

Reply:
i have made the changes and pushed to the repo